proto/pubsub/v1/params.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package pubsub.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/peggyjv/sommelier/v4/x/pubsub/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
If params is empty, is it worth adding?

Reply:
I'm not sure if it's strictly required, but cellarfees also started out with empty params and I feel like it's good to have the scaffolding in place.